modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -189,15 +189,14 @@ Library version => Kerberos 5 release 1.17
 ----
 --
 
-* Install `mod_auth_kerb` for Apache
+* Install handy command-line tools for Kerberos: +
+Note that `krb5-user` is not an actual requirement but it will provide handy command-line tools for Kerberos like to manipulate keytab files.
```

Comment:
```suggestion
Note that `krb5-user` is not an actual requirement, but it will provide handy command-line tools for Kerberos, such as the ability manipulate keytab files.
```

Reply:
Changes in `config_apps_sample_php_parameters` need to be done in core and then transported with a config-to-docs run to the admin docs. I will take care.